Description: Fixes bad #!/perl in l2hconf.pin
 This patch removes the first line "#!/perl" from l2hconf.pin
Author: Roland Stigge <stigge@antcom.de>

--- latex2html-2008-debian1.orig/l2hconf.pin
+++ latex2html-2008-debian1/l2hconf.pin
@@ -1,4 +1,3 @@
-#!/perl
 # LaTeX2HTML l2hconf.pm
 # $Id: l2hconf.pin,v 1.17 2002/06/15 22:46:36 RRM Exp $
 

